account_credit_control/views/credit_control_line.xml

```diff
@@ -23,6 +23,7 @@
                         </group>
                         <group>
                             <field name="date" readonly="state != 'draft'" />
+                            <field name="date_entry" />
```

Comment:
Why?

Reply:
This is a useful information, remember that there are 3 dates:
- Date (`date`): Date of the credit control line.
- Entry date (`date_entry`): The date of the journal item (`move_line_id.date`).
- Due date (`date_due`): Due date.

It is important to have in context all the dates because each one represents different things and will probably be different. Maybe a hidden optional field?